src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -829,6 +836,7 @@ private Future<Void> verifyDeleteAllowed(PoLine line, RequestContext requestCont
   }
 
   private Future<Void> createShadowInstanceIfNeeded(CompositePoLine compositePoLine, RequestContext requestContext) {
+    logger.info("createShadowInstanceIfNeeded:: Creating shadow instance if needed");
```

Comment:
we have full logs into inventoryInstanceManager.createShadowInstanceIfNeeded that is invoked under the hood.
We can add that log into if statement instead.